### DISCUSSION TITLE
How to find the admin id and password?

### DISCUSSION THREAD

**Discussion by shubh3ai** [codemation/easyauth#68] [Q&A]
I read the documentation and it says to search the logs. But I cannot find the password to login. Also, is there some easy way of setting admin id and password?

**Comment by codemation** [comment]
Hello @shubh3ai,

Its not yet documented, but before the 'admin' user is created, it is possible to set the `TEST_INIT_PASSWORD` environment variable, which will set the `admin` user password to the value, only if the `admin` user does not yet exist.  Probably best that you clear the current users table and start over using the new env variable or watching logs, then reset the password ASAP.
